server/routes/api/users.js

```diff
@@ -75,4 +76,78 @@ router.post("/login", (req, res) => {
   }
 });
 
+// @route GET /api/users/{id}
+// @desc Get a User object
+// @access Authenticated User can get own record
+router.get("/:id", (req, res) => {
+  let id = req.params.id;
+  User.findById(id, function(err, user) {
+    if (!user) {
+      res.status(404).send({ id: `User with id ${id} is not found` });
```

Comment:
This route currently in theory allows any user get any other user - instead you should use the JWT middleware and just return req.user

Reply:
Added. Thanks